src/GovUk.Education.ExploreEducationStatistics.Data.Processor.Tests/Services/DataImportServiceTests.cs

```diff
@@ -148,7 +149,8 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
                 var updated = await contentDbContext.DataImports.FindAsync(import.Id);
                 Assert.NotNull(updated);
                 Assert.Equal(1000, updated.RowsPerBatch);
-                Assert.Equal(10000, updated.TotalRows);
+                Assert.Equal(10000, updated.Rows);
+                Assert.Equal(5000, updated.TotalRows);
```

Comment:
This seems a little confusing. These values don't seem to correspond well with the parameters `totalRowCount` and `importableRowCount` passed to `service.Update`.

Would expect that:
- `totalRowCount` => `TotalRows`
- `importableRowCount` => `Rows`

However, it seems that the reverse is true. Perhaps the service method needs to be updated to have more appropriate parameters.

It also seems strange that the total rows would be less than rows, shouldn't total rows always be larger? That said, I'm just generally confused what the difference between rows, total rows even is. The terminology is a little confusing and could use some clarification (perhaps in a docblock or comment).

Reply:
That's correct, the reverse was true, and total rows was always less than rows which was the real total.

It really needed the underlying columns of `DataImport` renaming to correct this which I didn't want to do but I'm glad now you've raised it as it's justified the renaming a bit more!

https://github.com/dfe-analytical-services/explore-education-statistics/pull/3405 has renamed `TotalRows` -> `ImportedRows` and `Rows` -> `TotalRows`.

This means the service call now looks something like:

```
service.Update(import.Id,
    importedRows: 5000,
    totalRows: 10000,
    numBatches: 10,
    ...
);
```
and the assertions now correspond:

```
    Assert.Equal(5000, updated.ImportedRows);
    Assert.Equal(10000, updated.TotalRows);
````